examples/android/LuminaDemo/README.md

```diff
@@ -1,18 +1,20 @@
 # Instructions to re-create the setup
 
 - Create a new kotlin project in xcode
-- make sure you have aarch64-apple-ios target installed + simulator
+- make sure you have all the default android targets installed
 ```
-rustup target add aarch64-linux-android armv7-linux-androideabi i686-linux-android thumbv7em-none-eabihf wasm32-unknown-unknown x86_64-linux-android
+rustup target add aarch64-linux-android armv7-linux-androideabi i686-linux-android
```

Comment:
iirc we build for `thumbv7em-none-eabihf`

Reply:
This is a [bare-metal target](https://doc.rust-lang.org/nightly/rustc/platform-support/thumbv7em-none-eabi.html), so I'm pretty sure I had that installed because I was working on some arduinish projects in rust.